app/src/main/java/com/hyperion/ths/marvel_03/ui/heroinfo/HeroInfoViewModel.java

```diff
@@ -0,0 +1,38 @@
+package com.hyperion.ths.marvel_03.ui.heroinfo;
+
+import com.hyperion.ths.marvel_03.R;
+import com.hyperion.ths.marvel_03.data.model.Hero;
+import com.hyperion.ths.marvel_03.ui.BaseViewModel;
+import com.hyperion.ths.marvel_03.utils.Constant;
+import com.hyperion.ths.marvel_03.utils.navigator.Navigator;
+
+/**
+ * Created by ths on 01/06/2017.
+ */
+
+public class HeroInfoViewModel extends BaseViewModel {
+    private Hero mHero;
+    private Navigator mNavigator;
+
+    public HeroInfoViewModel(Hero hero, Navigator navigator) {
+        mHero = hero;
+        mNavigator = navigator;
+    }
+
+    public String getHeroName() {
+        return mNavigator.getActivity().getString(R.string.hero_name) + mHero.getName();
+    }
+
+    public String getHeroId() {
+        return mNavigator.getActivity().getString(R.string.hero_id) + String.valueOf(mHero.getId());
```

Comment:
WARNING: [UselessStringValueOf] No need to call String.valueOf to append to a string.
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/strings.html#UselessStringValueOf